### ISSUE TITLE
More Art and Info when skiping through playback as option.

### ISSUE THREAD

**Issue by QuickSwitchLK** [jurialmunkey/skin.arctic.fuse#390]
### Skin section

Widgets

### Feature description

Currently when skipping through on playback the following details are shown.

<img width="1225" alt="Screenshot 2024-01-05 151854" src="https://github.com/jurialmunkey/skin.arctic.fuse/assets/34122743/4e1914fe-16a1-44ba-bb70-1102c387ecb1">

Can we have a option that we can enable that shows the extra information you see when you press the enter key. ( See Image Below )

<img width="1233" alt="Screenshot 2024-01-05 151752" src="https://github.com/jurialmunkey/skin.arctic.fuse/assets/34122743/32d573c5-820e-429c-98b6-ae8d4504b265">

So to be clear, just the information above the time line. Clearlogo, Res Icon, Year

Also is there a way to change the period of time the OSD of playback is displayed, at the moment after a seek key is press it disappears after less than say a second?

### Why is this feature needed? Please justify and explain.

Mostly for eye candy to look good. Like the extra info. Why hide the nice away?

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by xyzfre** [comment]
![image](https://github.com/jurialmunkey/skin.arctic.fuse/assets/49699869/de511940-26db-4d59-92df-70bcc4a4961a)

**Comment by QuickSwitchLK** [comment]
Yep, have that enabled and in use,  but that is for when paused. I am requesting for changes for what it displayed on screen when seeking through the playimg media. Currently it is very basic, maybe for an important reason?

**Comment by jurialmunkey** [comment]
It's basic to avoid obscuring subtitles and the like. I had other people request that I remove clearlogo for seeking because it makes it harder to see the screen -- which I agreed with because it makes it harder to see what you're seeking to.

**Comment by QuickSwitchLK** [comment]
Thanks again for adding this back in. Much appreciated. Can I extend the time it is shown on the screen after the final button press? Currently, the only solution I have found is maxing out the player skip delay to 1500ms, but this has its issues lol.

**Comment by jurialmunkey** [comment]
No it's not currently configurable. There's a much better way in omega to handle the display after seek timing - - I'm basically waiting on omega reaching release so I can drop the nexus back porting and not worry about adding features like that which will completely break the skin in nexus.
